sphinx/templates/quickstart/make.ps1_t

```diff
@@ -0,0 +1,198 @@
+# ==========================================================================
+[CmdletBinding()]
+param(
+  # Make all targets
+  [switch]$all =        $false,
+  # Make standalone HTML files
+  [switch]$html =       $false,
+  # Make HTML files named index.html in directories
+  [switch]$dirhtml =    $false,
+  # Make a single large HTML file
+  [switch]$singlehtml = $false,
+  # Make pickle files
+  [switch]$pickle =     $false,
+  # Make json files
+  [switch]$json =       $false,  
+  # Check all external links for integrity
+  [switch]$linkcheck =  $false,
+  # Clean up directory before running Sphinx.
+  [switch]$clean =      $false,
+  # Show the embedded help page(s).
+  [switch]$help =       $false
+)
+
+# resolve paths
+$WorkingDir =     Get-Location
+$SphinxRootDir =  Convert-Path (Resolve-Path ($PSScriptRoot))
+
+# set default values
+$EnableVerbose =  $PSCmdlet.MyInvocation.BoundParameters["Verbose"]
+$EnableDebug =    $PSCmdlet.MyInvocation.BoundParameters["Debug"]
+if ($EnableVerbose -eq $null)  { $EnableVerbose =  $false }
+if ($EnableDebug   -eq $null)  { $EnableDebug =    $false }
+if ($EnableDebug   -eq $true)  { $EnableVerbose =  $true  }
+
+# Display help if no command was selected
+$Help = $Help -or (-not ($all -or $html -or $dirhtml -or $singlehtml -or $pickle -or $linkcheck -or $clean -or $help))
+
+function Exit-Script
+{ <#
+    .PARAMETER ExitCode
+    ExitCode of this script run
+  #>
+  [CmdletBinding()]
+  param([int]$ExitCode = 0)
+  
+  # restore environment
+  # rm env:GHDL -ErrorAction SilentlyContinue
+  
+  cd $WorkingDir
+  
+  # unload modules
+  # Remove-Module precompile -Verbose:$false
+
+  # exit with exit code
+  exit $ExitCode
+}
+
+if ($Help)
+{ Get-Help $MYINVOCATION.InvocationName -Detailed
+  Exit-Script
+}
+
+if ($All)
+{ $clean =      $true
+  $html =       $true
+  $dirhtml =    $true
+  $singlehtml = $true
+}
+
+$SphinxBuild =    if (-not $env:SPHINXBUILD) { "sphinx-build" } else { $env:SPHINXBUILD }
+$BuildDir =       "$SphinxRootDir\{{ rbuilddir }}"
+$SourceDir =      "$SphinxRootDir\{{ rsrcdir }}"
+# $BuildDir =       "$SphinxRootDir\_build"   # for local testing, can be removed in the future
+# $SourceDir =      "$SphinxRootDir\."        # for local testing, can be removed in the future
+$AllSphinxOpts =  "-d $BuildDir\doctrees ${env:SPHINXOPTS} $SourceDir"
+$I18NSphinxOpts = "${env:SPHINXOPTS} $SourceDir"
+# ------------------------------------------------------------------------------
+# TODO: add paper options
+# ------------------------------------------------------------------------------
+# if NOT "%PAPER%" == "" (
+#   set ALLSPHINXOPTS=-D latex_paper_size=%PAPER% %ALLSPHINXOPTS%
+#   set I18NSPHINXOPTS=-D latex_paper_size=%PAPER% %I18NSPHINXOPTS%
+# )
+
+# Check if sphinx-build is available and fallback to Python version if any
+# ------------------------------------------------------------------------------
+# TODO: add testings if sphinxbuild can be called and/or is installed
```

Comment:
The `make.bat` version has a text if Sphinx is installed and falls back to use `python -m sphinx`. Should I reimplement this behavior for the PowerShell version too?